core/src/main/java/org/apache/carbondata/core/util/DataFileFooterConverter.java

```diff
@@ -123,6 +123,25 @@ private BlockletInfo getBlockletInfo(
   }
 
   @Override public List<ColumnSchema> getSchema(TableBlockInfo tableBlockInfo) throws IOException {
-    return null;
+    FileHolder fileReader = null;
```

Comment:
Is this change related to this PR? Can you add comment to describe the intention of this function in its abstract interface?

Reply:
yes, it is part of this PR. We need to get columnschema